backend/src/modules/match-module/presentation/rest-api/MatchRouter.ts

```diff
@@ -37,6 +39,28 @@ export function matchRouter(
     );
   };
 
+  const postEndMatch = async (request: Request, response: Response) => {
+    const { matchId } = request.params;
+    const requestBody: PostEndMatchBody = request.body;
+    const commandResult = await commandPublisher.execute(
+      new EndMatch({
+        matchId: matchId,
+        winnerId: requestBody.winnerId,
+      }),
+    );
+    return commandResult.process(
+      () =>
+        response
+          .status(StatusCodes.OK)
+          .json({
+            matchId: matchId,
```

Comment:
To też niepotrzebne, tak samo jak w przypadku tworzenia meczu. 
To łatwo rozkminić - response zwraca Ci tylko to o czym nie wiesz. A tutaj to wszystko masz z argumentów :) 
Zazwyczaj jest to tylko ID przy tworzeniu obiektu lub informacja o powodzeniu / błędach operacji. 

Czasem się jeszcze zwraca zaktualizowany stan encji, ale raczej nie polecam takiej praktyki.